packages/.eslintrc.json

```diff
@@ -40,8 +40,6 @@
         "no-unmodified-loop-condition": "error",
         "no-unneeded-ternary": "error",
         "no-useless-concat": "error",
-        "no-useless-rename": "error",
```

Comment:
plz DO NOT MODIFY `.eslintrc`

Reply:
@Randolph314 looks like you didn't sync correctly with the base branch. You're reverting many commits.

Reply:
@Jack-Works so I need to merge develop branch?